src/welcome/showWelcomeComponent.tsx

```diff
@@ -0,0 +1,12 @@
+import { wasUsed } from './constants';
+
+export class FirstUsing {
+  wasUsed = (): boolean => {
+    const keyLocalStorage = localStorage.getItem(wasUsed);
+    return Boolean(keyLocalStorage);
+  };
+
+  markAsUsed = (): void => {
+    localStorage.setItem(wasUsed, 'false');
+  };
+}
```

Comment:
instead of instantiating the objects of `FirstUsing` class in every place, we could add on line 14:

```
export const firstEntry = new FirstUsing();
```

And then import this `firstEntry` wherever you like in the app.

So you could then remove line 6 in `App.tsx`

Reply:
corrected